python/TestHarness/TestHarness.py

```diff
@@ -377,6 +380,33 @@ def appendRecoverableTests(self, testers):
         testers.extend(new_tests)
         return testers
 
+    def appendReguritateTests(self, testers):
```

Comment:
Would be pretty awesome if this works reliably. We'll have to see!

Reply:
If we address the bullets in this issue I think we can do this.